.gitignore

```diff
@@ -13,6 +13,11 @@ dist
 dist-ssr
 *.local
 
+.env*
```

Comment:
I don't recommend doing this. because we need something like `.env.template` to know about our environment variables